compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -57,7 +59,8 @@ enumerateMatmulTilesVMVX(EncodingUser user, ExecutableTargetAttr target) {
 // For narrow-{M,N} cases, this only enumerates on narrow M. The narrow-N cases
 // are handled by transposition in chooseMatmulTile.
 static SmallVector<TileMxNxK>
-enumerateMatmulTileRiscv32(EncodingUser user, ExecutableTargetAttr target) {
+enumerateMatmulTileRiscv32(linalg::ContractionDimensions cDims,
```

Comment:
I just found that we don't need `ContractionDimensions` in the method at all? Maybe we can drop it from arguments?

Same for other backends. I think it is only needed by VMVX backend.